index.html

```diff
@@ -0,0 +1,145 @@
+            <h1>"Bonjour Money Badass!"</h1>
+            <h2>Financial Freedom Roadmap 2023</h2>
+        </div>
+        <div class="intro-text">
+            <p class="">
+                A fulfilling convention for all to learn the best way to secure and lead a life in financial freedom and early retiment. The launch will be on May 24th and 25th if you desire to be a part of the road to FIRE please do join us!
+            </p>
+            <div class="details">
+                <h2>2023.05.24(THU) - 25(FRI)</h2>
+                <h4>@Radisson Blu Hotel</h4>
+            </div>
+        </div>
+    </section>
+    <section class="program-section">
+        
+        <div>
+            <h3>Main Program</h3>
+        </div>
+        <div class="program-display">
+            <ul>
+                <li>
+                    <img src="assets/lecture.png" alt="">
+                    <h4>Lecture</h4>
+                    <p>Movement that promotes the idea of saving and investing aggressively in order to reach a state of financial independence, where one's passive income is sufficient to cover their living expenses, at a young age</p>
+                </li>
+                <li>
+                    <img src="assets/presentation.png" alt="">
+                    <h4>CC Exhibition</h4>
+                    <p>F.I.R.E movement is to achieve financial independence and the freedom and flexibility it brings, in order to pursue other passions and interests, or simply to enjoy a more leisurely and fulfilling life</p>
+                </li>
+                <li>
+                    <img src="assets/forum.png" alt="">
+                    <h4>Forum</h4>
+                    <p>Investing aggressively in order to reach a state of financial independencein order to pursue other passions and interests, or simply to enjoy a more leisurely and fulfilling life</p>
+                </li>
+                <li>
+                    <img src="assets/chat.png" alt="">
+                    <h4>Workshop</h4>
+                    <p>The idea behind F.I.R.E. is to have the freedom and flexibility to live one's life on one's own terms, without being financially dependent on a job or traditional sources of income!</p>
+                </li>
+                <li>
+                    <img src="assets/fire-round-button.png" alt="">
+                    <h4>F.I.R.E Ignite</h4>
+                    <p>Master the ability to retire at an early age, typically in one's 30s or 40s, by saving and investing a high percentage of one's income, living frugally, and building passive income streams.</p>
+                </li>
+            </ul>
+        </div>
+            <button type="button" class="orange-btn"> Join The Financial Freedom Community
+            </button>
+        
+    </section>
+
+    <section class="speakers-section">
+        <h3>Featured Speakers</h3>
+        <ul class="speakers-list">
+            <!-- Dynamically added by Javascript -->
+        </ul>
```

Comment:
- [ ] Please update the size of the images in this section. They look a bit smaller on the desktop version when compared to the design. 🤝️

**Design**
![Screenshot from 2023-01-04 19-16-02](https://user-images.githubusercontent.com/17243325/210610326-96771dca-eeac-4695-9b75-3e03a44a4946.png)

**Your work**
![Screenshot from 2023-01-04 19-16-19](https://user-images.githubusercontent.com/17243325/210610337-088bfe2d-bdc9-4594-ae81-f57c470b6eab.png)